### PR TITLE
Jake Rambold Timed Friday Challenge

### PR DESCRIPTION
**Pull request by jrambold**

### CODE REVIEW

**Comment by jrambold** [comment]
First 2 user stories finished

File: app/controllers/books_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class BooksController < ApplicationController
+  def show
+    @book = Book.find(params[:id])
+    @reviews = @book.reviews
+    @rating = @book.average
```
Comment (AliSchlereth) [inline app/controllers/books_controller.rb]: You generally want to pass as few instance variables to the view as possible. In this scenario you call both reviews and average on a book instance which is getting passed to the view. I'd expect to call @book.reviews and @book.average in the view and leave off @reviews and @rating.

File: spec/features/user_visits_book_show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    visit book_path(book)
+
+    expect(page).to have_content(review.body)
+    expect(page).to have_content(review.rating)
+    expect(page).to have_content(user.id)
+
+  end
+
+  scenario "I also see the rating 1-5 that the user gave the book" do
```
Comment (AliSchlereth) [inline spec/features/user_visits_book_show_spec.rb]: You built it correctly, but this name is off. It is the average rating across all users.

**Comment by AliSchlereth** [comment]
@jrambold You completed 2 out of 6 user stories. This is significantly below expectations. If you have not already, I highly recommend you re-do this challenge, making sure to work through the end.

**Pull request closed without merging**